js/onoffice-apply-thousand-separator.js

```diff
@@ -0,0 +1,58 @@
+onoffice_setting = onoffice_setting || [];
+jQuery(document).ready(($) => {
+    const thousandSeparatorFormat = onoffice_setting.thousand_separator_format || '';
+
+    const getSeparators = (separatorFormat) => ({
+        thousandSeparator: separatorFormat === 'comma-separator' ? ',' : '.',
+        decimalSeparator: separatorFormat === 'comma-separator' ? '.' : ','
```

Comment:
In https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/721#issuecomment-1966093746 you mention that you did not implement decimal separator. 
Does onoffice-apply-thousand-separator.js need some refactoring ?